README.md

```diff
@@ -24,37 +24,52 @@ python3 -m pip install --extra-index-url https://artifacts.luxonis.com/artifacto
  - cmake >= 3.4
  - C++14 compiler (clang, gcc, msvc, ...)
  - Python3
+ - Submodule dependencies with Git
 
 Along these, dependencies of depthai-core are also required
 See: [depthai-core dependencies](https://github.com/luxonis/depthai-core#dependencies)
 
 
 ### Building
 
+The first time you build, the repository submodules need be initialized:
+```
+git submodule update --init --recursive
```

Comment:
This is required and recommended every single time.
Additionally, can be automatically set on checkout (recommended):
Global git option:
`git config --global submodule.recurse true`

local to this repo:
`git config submodule.recurse true`

Reply:
What do you think of a side note or tip, then? I personally prefer not requesting people to change their Git settings.

```suggestion
git submodule update --init --recursive

# Tip: You can ask Git to do that automatically:
git config submodule.recurse true
```

Reply:
Yeah, it meant to be a side note 👍🏻